modules/e2ee/ManagedKeyHandler.js

```diff
@@ -167,6 +184,28 @@ export class ManagedKeyHandler extends KeyHandler {
         this.e2eeCtx.setKey(id, key, index);
     }
 
+    /**
+     * Handles the SAS ready event.
+     *
+     * @param {string} pId - The participant ID.
+     * @param {Uint8Array} sas - The bytes from sas.generate_bytes..
+     * @private
+     */
+    _onParticipantSasReady(pId, sas) {
+        this.conference.eventEmitter.emit(JitsiConferenceEvents.E2EE_SAS_READY, pId, sas);
+    }
+
+    /**
+     * Handles the SAS completed event.
+     *
+     * @param {string} pId - The participant ID.
+     * @param {boolean} success - Wheter the verification was succesfull.
+     * @private
+     */
+    _onParticipantSasCompleted(pId, success) {
```

Comment:
The event should just say the participant is verified with success or not, but SAS completed sounds too low level here.